### PR TITLE
fix/PP-171

### PR DESCRIPTION
**Pull request by franciscotobar**
## What

- Fixing feature/PP-171

## Why

- Integration testing were not being successful

### CODE REVIEW

File: src/RelayProvider.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ import { AccountKeypair } from './AccountManager';
 import { RelayEvent } from './RelayEvents';
 import { Address } from './types/Aliases';
 import { toBN, toChecksumAddress, toHex } from 'web3-utils';
+import { Transaction } from 'ethereumjs-tx';
```
Comment (antomor) [inline src/RelayProvider.ts:22]: Could you please remove this if not necessary?

File: src/RelayProvider.ts [hunk 2]
```diff
@@ -237,6 +242,73 @@ export default class RelayProvider implements HttpProvider {
         return this.executeRelayTransaction(transactionDetails);
     }
 
+    async processTransactionReceipt(
```
Comment (antomor) [inline src/RelayProvider.ts:245]: probably this method should be private, shouldn't it?

**Review by antomor — CHANGES_REQUESTED** [review]

File: src/RelayProvider.ts (hunk 1 above)
Comment (franciscotobar) [reply src/RelayProvider.ts]: done

File: src/RelayProvider.ts (hunk 2 above)
Comment (franciscotobar) [reply src/RelayProvider.ts]: done

**Review by antomor — APPROVED** [review]
LGTM

**Pull request merged**